src/content/docs/en/guides/cms/contentful.mdx

```diff
@@ -216,16 +219,18 @@ Feel free to add as many blog posts as you want, then switch to your favorite co
 
 Create a new interface called `BlogPost` and add it to your `contentful.ts` file in `src/lib/`. This interface will match the fields of your blog post content type in Contentful. You will use it to type your blog post entries response.
 
-```ts title="src/lib/contentful.ts" ins={2,4-10}
-import contentful from "contentful";
-import type { Document } from "@contentful/rich-text-types";
+```ts title="src/lib/contentful.ts" ins=", { EntryFieldTypes }" ins={3-12}
```

Comment:
```suggestion
```ts title="src/lib/contentful.ts" ins="{ EntryFieldTypes }" ins={3-12}
```
Just checking that you *meant* to show that you needed to add the comma, too! Judgement call, I'm happy either way as long as it's intentional, and wanted to make it clear that I SAW this!  :smile:

Reply:
yep! this is intentional, but I'm open to feedback on this. for consistency, is it a good idea to adopt a pattern of including/excluding this?

Reply:
I'm good with this as-is for now!

I only wanted to point it out to make sure that it was known that I DID see it and I DID review it! (Because it might have looked weird on first glance to someone else.) Confirming that I approve!